.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.formatOnSave": true
```

Comment:
Need to remove ide settings from the source
and update .gitignore to exclude .vscode